patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-top-ToolbarPhone.java.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+index 9610f24f8a43e081fc260ab4be93718488fe624f..3bb90ff4a4ca2082925021c0e0799d7299c55fc9 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+@@ -88,7 +88,7 @@ import java.util.List;
+ /**
+  * Phone specific toolbar implementation.
+  */
+-public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, OnClickListener,
++public class ToolbarPhone extends BraveToolbarLayout implements Invalidator.Client, OnClickListener,
+                                                            NewTabPage.OnSearchBoxScrollListener,
+                                                            TabCountObserver {
+     /** The amount of time transitioning from one theme color to another should take in ms. */
+@@ -426,10 +426,12 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+     /**
+      * Set the background color of the location bar to appropriately match the theme color.
+      */
+-    private void updateModernLocationBarColor(int color) {
++    @Override
++    protected void updateModernLocationBarColor(int color) {
+         if (mCurrentLocationBarColor == color) return;
+         mCurrentLocationBarColor = color;
+         mLocationBarBackground.setColorFilter(color, PorterDuff.Mode.SRC_IN);
++        super.updateModernLocationBarColor(color);
+     }
+ 
+     /**
+@@ -546,6 +548,7 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+                         .notifyEvent(EventConstants.PARTNER_HOME_PAGE_BUTTON_PRESSED);
+             }
+         }
++        super.onClick(v);
+     }
+ 
+     @Override
+@@ -719,7 +722,8 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+      * @return The right bounds of the location bar after accounting for any visible left buttons.
+      */
+     private int getBoundsAfterAccountingForRightButtons() {
+-        return Math.max(mToolbarSidePadding, mToolbarButtonsContainer.getMeasuredWidth());
++        return Math.max(mToolbarSidePadding, 
++            super.getBoundsAfterAccountingForRightButtons(mToolbarButtonsContainer));
+     }
+ 
+     private void updateToolbarBackground(int color) {
+@@ -2121,6 +2125,9 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+         } else {
+             populateUrlClearFocusingAnimatorSet(animators);
+         }
++        populateUrlAnimatorSet(hasFocus, URL_FOCUS_TOOLBAR_BUTTONS_DURATION_MS,
```

Comment:
try to keep patches to one line even if it violates lint character line limits. We don't run lint on these files anyway